lib/index.coffee

```diff
@@ -21,6 +22,9 @@ lint = (editor) ->
   convertOldConfig()
   command = atom.config.get(COMMAND_CONFIG_KEY).split(/\s+/).filter((i) -> i)
     .concat(DEFAULT_ARGS, filePath = editor.getPath())
+  if atom.config.get(DISABLE_CONFIG_KEY) is true
+    config = helpers.find(filePath, '.rubocop.yml')
+    return [] if config is null or not atom.project.rootDirectories.some (p) -> config.startsWith(p.path)
```

Comment:
Yes. I've gone a bit back and forth on whether that's a good idea.
I think I'll remove it, since the default behavior for RuboCop is to look all the way to the root directory.